.zappr.yaml

```diff
@@ -0,0 +1,13 @@
+approvals:
+  groups:
+    zalando:
+      minimum: 2
+      from:
+        orgs:
+          - "zalando"
+
+  # Allow last committer / PR creator to approve as well.
+  # This will probably become the default in zappr.
+  ignore: none
+  
+X-Zalando-Team: "Hollywood"
```

Comment:
I assumed those `X-Zalando-` properties were for the internal GHE only. Do we need them also for the open source projects?

Reply:
I'm not sure. I saw that all the other project have this as well